src/js/helpers/qajaxWrapper.js

```diff
@@ -0,0 +1,76 @@
+var qajax = require("qajax");
+
+var qajaxWrapper = function (options) {
+  var response = {
+    status: null,
+    body: null
+  };
+  // Translate Oboe options to qajax equivalent
+  var oboeDictionary = {
```

Comment:
Yeah I had a similar thought... I actually think it's only oboe doing it differently, so maybe we should refactor all clients to use the typical "jQuery-like" format and let oboeWrapper do the conversion. What do you think?